service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/SpecimenMapperTest.java

```diff
@@ -1,4 +1,4 @@
-package uk.nhs.adaptors.gp2gp.ehr.mapper;
```

Comment:
Minor change to move the `SpecimenMapperTest` to the diagnosticreport folder within the unit tests directory (similar to the `ObservationMapperTest`)